config.toml

```diff
@@ -461,3 +461,11 @@ languagedirection = "ltr"
 time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
+
```

Comment:
At some point, the `fa` language param should be placed alphabetically into the list of languages?

Reply:
I've changed the order.